dev/NavigationView/NavigationView.xaml

```diff
@@ -403,11 +447,13 @@
                                         BorderThickness="0,0,1,0">
                                         <Grid.RowDefinitions>
                                             <RowDefinition Height="Auto"/>
-                                            <RowDefinition Height="0"/> <!-- above button margin + back button space -->
+                                            <RowDefinition Height="0"/>
+                                            <!-- above button margin + back button space -->
                                             <RowDefinition x:Name="PaneContentGridToggleButtonRow" Height="Auto" MinHeight="{StaticResource NavigationViewPaneHeaderRowMinHeight}"/>
                                             <RowDefinition Height="Auto"/>
                                             <RowDefinition Height="Auto"/>
-                                            <RowDefinition Height="8"/> <!-- above list margin -->
+                                            <RowDefinition Height="8"/>
+                                            <!-- above list margin -->
```

Comment:
Move this in front of the row since the comment is referring to the height 8 definition.

Reply:
huh this must've happened during some weird merging. Reverted.